client/src/pages/Landing/index.tsx

```diff
@@ -1,10 +1,92 @@
+          <div className='HeaderContainer'>
+            <div className='HeaderLeftContainer'>
+              <a href="http://localhost:3000/">
+              <img src={require('../../images/logo.svg').default} alt="logo" width="32" height="32"/>
+              </a>
+            </div>
+            <div>
+              <Button color="inherit" onClick={() => (window.location.href = "http://localhost:3000/signin")}>Login</Button>
+              <Button
+                id='github-button'
+                color='inherit'
+                onClick={() => (window.location.href = 'https://github.com/roccopolimen/nextround')}
+              >
+                <GithubIcon></GithubIcon>
+              </Button>
+            </div>
+          </div>
+        </AppToolbar>
+      </AppBar>
+    </Slide>
+    {/* content */}
+      <Grid container direction='column' justify='center' alignItems='center' spacing={2}>
+        {/* top grid */}
+        <Grid container item sm={12} lg={9} justify='center' alignItems='center' spacing={3}>
+          {/* body 1 left */}
+          <Grid item sm={12} lg={6}>
+            <Slide direction='down' in={true} timeout={1000}>
+              <div className='HomePageContainer'>
+                <br/>
+                <br/>
+                <br/>
+                <Typography variant='h3'>NextRound</Typography>
+                <br/>
+                <Typography variant='subtitle1'>
+                  Track your job applications all in one place with NextRound’s unique features that will help you land the job of your dreams! 
+                  Ditch those spreadsheets; keep all the important information and upcoming events for each job application on site with a user 
+                  experience tailored for the job seeker.
+                </Typography>
+                <br/>
+                <div className="cta-button"> 
+                    {/* todo why buttons not purple */}
+                    <Button variant="contained" color="primary" id="signup" onClick={() => (window.location.href = "http://localhost:3000/signup")}>Sign up</Button>
+                    <Button variant="outlined" color="primary" id="signin" onClick={() => (window.location.href = "http://localhost:3000/signin")}>Log in</Button>
+                </div>
+              </div>
+            </Slide>
+          </Grid>
+          {/* right */}
+          <Grid item sm={12} lg={6}>
+            {/* TODO featured image here */}
+          </Grid>
+        </Grid>
 
-    return (
-        <div>Hello World</div>
-    );
+        {/* bottom grid */}
+        <Grid container item sm={12} lg={9} justify='center' alignItems='center' spacing={3}>
+          {/* body 2 left */}
+          <Grid item sm={12} lg={6}>
+            <Slide in={true} direction='up' timeout={2000}>
+              <div className='HomePageContainer'>
+                <Typography variant='h5'>Track Applications for Top Companies</Typography>
+                <div className="logos">
```

Comment:
The logo spacing and all doesn't look great, especially when resizing (as you noted), but I don't see a dire need for adjustment at this stage; we'll just keep it in mind.